code/__DEFINES/traits.dm

```diff
@@ -42,7 +43,8 @@
 #define TRAIT_DEPRESSION			"depression"
 #define TRAIT_JOLLY				"jolly"
 #define TRAIT_NOCRITDAMAGE		"no_crit"
-
+#define TRAIT_NOSLIPWATER		"noslip_water"
+#define TRAIT_NOSLIPALL			"noslip_all"
```

Comment:
more unused traits?

Reply:
NOSLIPWATER is used in water cookies and the blue stabilized extract. NOSLIPALL is here for future use in case someone wishes to add an effect that causes it.